### ISSUE TITLE
confirmation 

### ISSUE THREAD

**Issue by Mohammedsaied89** [flennerhag/mlens#138]
I want to know the data flow ( or diagram) of the superlearner stacking in mlens package.
If I have dataset so I split it to training and testing then I use the kfolds in the training set then I use the Predictions of base models to train the next level so what about the test set
